### DISCUSSION TITLE
如何在左下角头像添加一个按钮

### DISCUSSION THREAD

**Discussion by Jixiangup** [umijs/umi#11576] [Q&A]
<img width="492" alt="image" src="https://github.com/umijs/umi/assets/75556346/534502ac-df7e-4125-8da0-b01a10e2662d">

如图 我想在这里加一个按钮 比如 绑定租户 个人设置之类的按钮我应该怎么加

**Comment by fz6m** [comment]
这个展开菜单没法自定义，但是可以完全重写这部分，在 `src/app.tsx` 里通过 layout 的 `rightRender` 进行重写：

```ts
// app.tsx

export const layout = () => {
  return {
    rightRender: (initialState, setInitialState, runtimeConfig) => {
      return <ReactNode />
    }
  }
}
```

可以参考 `src/.umi/plugin-layout/rightRender.tsx` 里面的逻辑 copy 代码自定义自己的逻辑。
